fragments/k8s/core/bundle.yaml

```diff
@@ -9,9 +9,8 @@ applications:
     to:
       - "0"
     options:
-      channel: 1.23/edge
+      channel: 1.24/edge
     expose: true
-    constraints: "cores=2 mem=4G root-disk=16G"
```

Comment:
removed the charm constraint since these are defined in the `machines` stanza below.